docs/docs-site/content/developer/development/_index.md

```diff
@@ -820,8 +820,6 @@ A Pypi token will be needed. For doing a test release https://test.pypi.org/proj
     rm -rf dist && python3 -m build && python3 -m twine upload --repository testpypi dist/*
     python3 -m pip install --index-url https://test.pypi.org/simple/ gethue --upgrade
 
-Read more on [Python packaging](https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives).
```

Comment:
why we are removing this link? is this not working or not relevant?

Reply:
Its giving 403 error